src/arrayMethodSort.js

```diff
@@ -11,17 +11,20 @@ function applyCustomSort() {
       comparator = (a, b) => a.toString() > b.toString();
     }
 
-    let tmpValue;
+    let tmpValue, isExchange;
```

Comment:
You can initialize the variable `tmpValue` directly in your  `for` loop. It's needed only there and on each iteration, it has different values. It is not needed outside the loop